### DISCUSSION TITLE
New ship weapon: turret harpoon gun

### DISCUSSION THREAD

**Discussion by 8on8er** [FakeFishGames/Barotrauma#12136] [Ideas and suggestions]
As the title suggests:

My idea is a harpoon turret that functions similarly to the thalamus flesh harpoon. It would go on large turret hardpoints. You would be able to craft various harpoons which could be loaded like a railgun shell. If the harpoon snaps due to a fast or large target you would be able to retrieve the harpoon from the body of the creature.

I would believe for example some fast moving creatures would never be able to be harpooned or unable to keep the tether with just a standard steel harpoon.

You could also create poison harpoons using the doctor which would make the poison perks more valid which are currently underwhelming.

My main point being I think it would be an unreal gameplay experience to be able to for example harpoon a hammerhead slowing it down while a secondary coil gun murders it. I don't know the coding involved but I imagine if a latcher can drag a sub with its tongue and a thalamus sub can drag your sub that it might be possible?

I also think this could revolutionise pirate missions too. Finally you would be able to have a quick means of boarding the submarine.

Let me know your thoughts.

**Comment by pantherpusher** [comment]
it'd need code to tell the AIs that get tethered that they should try to move away from the submarine instead of trying to attack it. Lots of things relentlessly try to get as close to the submarine as they possibly can.

**Reply by Hazmat616** [reply]
Perhaps that's just a tradeoff. The main appeal for me is it would allow a greater yield of loot from monsters as they won't all sink to the bottom. Although plenty of monsters probably should at least have a random chance to fight or flight. Otherwise make only the angrier or mindless ones still charge.

**Comment by Hazmat616** [comment]
This would also create new mission potential requiring you bring in a large creature back to the station you got the mission at. The tether would not expire after the target is dead but could still be broken so need to reapply it.

**Comment by iridescent-lightning** [comment]
Barotrauma Die Hard has made one. You can even tether your ship to a cliff
